shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -625,14 +753,11 @@ - (instancetype)init {
       _smartQuotesType = UITextSmartQuotesTypeYes;
       _smartDashesType = UITextSmartDashesTypeYes;
     }
+    _selectionRects = @[];
```

Comment:
This is a potential crash, it should be `_selectionRects = [@[] retain]`;  or `_selectionRects = [[NSArray alloc] init]`.  The literal is on the autorelease pool and can be deleted before you access it.